pennylane/interfaces/tfe.py

```diff
@@ -201,7 +201,6 @@ def circuit(phi, theta):
 
 import numpy as np
 import tensorflow as tf
-import tensorflow.contrib.eager as tfe # pylint: disable=unused-import
```

Comment:
How come this line was removed? Was this import not being used?

Reply:
I actually don't know, that was nothing I meant to do. Let me add it again.